Library/.rubocop.yml

```diff
@@ -189,14 +189,7 @@ Naming/MethodParameterName:
       - AllowedNames
   AllowedNames:
     [
-      "a",
-      "b",
-      "c1",
-      "c2",
-      "e",
-      "f",
-      "o",
-      "pr",
+      "pr", # TODO: Remove if https://github.com/rubocop/rubocop/pull/11690 is merged or we change the variable names.
```

Comment:
Might want to move f to another PR so the tap syntax changes don't block this one?